docs/developers/built-in-functions.md

```diff
@@ -115,3 +118,69 @@ already covered [here](/developers/account-storage).
 
 Most of the ESDT and NFT related built-in function are already described in the [ESDT](/developers/esdt-tokens/) and
 [NFT](/developers/nft-tokens) sections.
+
+[comment]: # (mx-context-auto)
+
+## **SetGuardian**
+
+`SetGuardian` is used to set a guardian for a given address. 
+
+```rust
+SetGuardianTransaction {
+    Sender: <sender address>
+    Receiver: <sender address>
+    Value: 0
+    GasLimit: 1_000_000
+    Data: "SetGuardian" +
+          "@" + <guardian address in hexadecimal encoding> +
+          "@" + <service ID in hexadecimal encoding>
+}
+```
```

Comment:
add
`_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._`